util/fhir.ts

```diff
@@ -6,6 +6,8 @@ import { parsedPrimaryCodePaths } from './primaryCodePaths';
 import _ from 'lodash';
 import { ReferencesMap } from './referencesMap';
 
+const fhirpath = require('fhirpath');
```

Comment:
I think we can change this to `import fhirpath from 'fhirpath';` for consistency